__tests__/migration.test.ts

```diff
@@ -0,0 +1,142 @@
+import { createAgent, TAgent, IDIDManager, IResolver, IKeyManager, IDataStore } from '@veramo/core'
+import { createConnection, Connection } from 'typeorm'
+import { DIDResolverPlugin } from '../packages/did-resolver'
+import { EthrDIDProvider } from '../packages/did-provider-ethr'
+import { WebDIDProvider } from '../packages/did-provider-web'
+import { KeyDIDProvider } from '../packages/did-provider-key'
+import { DIDComm, IDIDComm } from '../packages/did-comm'
+import { KeyManagementSystem, SecretBox } from '../packages/kms-local'
+import {
+  Entities,
+  IDataStoreORM,
+  DataStore,
+  DataStoreORM,
+  KeyStore,
+  DIDStore,
+  PrivateKeyStore,
+  migrations,
+} from '../packages/data-store'
+import { Resolver } from 'did-resolver'
+import { getResolver as ethrDidResolver } from 'ethr-did-resolver'
+import { getResolver as webDidResolver } from 'web-did-resolver'
+import { getDidKeyResolver } from '../packages/did-provider-key'
+import { KeyManager } from '../packages/key-manager'
+import { DIDManager } from '../packages/did-manager'
+import { FakeDidProvider, FakeDidResolver } from './utils/fake-did'
+import fs from 'fs'
+
+jest.setTimeout(30000)
+
+const databaseBeforeFile = __dirname + '/fixtures/local-database-before-3.0.sqlite'
+const databaseFile = __dirname + '/migrated.database.sqlite'
+const infuraProjectId = '5ffc47f65c4042ce847ef66a3fa70d4c'
+const secretKey = '29739248cad1bd1a0fc4d9b75cd4d2990de535baf5caadfdf8d8f86664aa830c'
```

Comment:
Is this meant to be a secretKey with infura? If it is and it's actually meant to be secret I'd suggest passing it in as an environment var through a config file so it's not being revealed in commit history

Reply:
This key is meant to decrypt some data in the static DB used during tests.
That is a very good observation. I will try to incorporate it for best practice.